packages/modelence/src/config/server.ts

```diff
@@ -41,7 +41,7 @@ let isInitialized = false;
  * ```
  */
 export function getConfig(key: ConfigKey) {
-  return config[key]?.value;
+  return config[key]?.value ?? configSchema[key]?.default;
 }
```

Comment:
@omegascorp should we do similar change for the client side as well ? in "/src/config/client.ts"

Reply:
good then I guess we can merge this